src/OrbitGl/ManualInstrumentationManager.h

```diff
@@ -28,6 +28,7 @@ class ManualInstrumentationManager {
   void AddAsyncTimerListener(AsyncTimerInfoListener* listener);
   void RemoveAsyncTimerListener(AsyncTimerInfoListener* listener);
   void ProcessAsyncTimer(const orbit_client_protos::TimerInfo& timer_info);
+  void ProcessAsyncTimerDeprecated(const orbit_client_protos::TimerInfo& timer_info);
```

Comment:
Move this up one line or move down the definition in the .cpp to have the same order.

Reply:
done, thanks.